### PR TITLE
[CDF-13342] Fix doc typo

### PR DESCRIPTION
**Pull request by aurlien**
## Description
Update typos in the documentation examples for templates. 

## Checklist:
- [ ] Tests added/updated.
- [x] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change. 
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring. 
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) per [semantic versioning](https://semver.org/).

### CODE REVIEW

**Review by me-ydv-5 — APPROVED** [review]

**Comment by me-ydv-5** [comment]
this won't directly start reflecting in the docs, there's some manual step (automatic but broken atm) that generates json docs and then you copy that json docs to service-contract repo. if it's urgent i can do it today.

**Comment by aurlien** [comment]
Thanks, @me-ydv-5! I don't think it's urgent to get it out. 

/cc @Tapped

**Comment by erlendvollset** [comment]
The pysdk docs will be automatically updated through the readthedocs webhook though.

File: cognite/client/_api/templates.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def graphql_query(self, external_id: str, version: int, query: str) -> GraphQlRe
                 >>>        }
                 >>>    }
                 >>>    '''
```
Comment (nayang-cognite) [inline cognite/client/_api/templates.py:49]: User also mentioned missing some []’ for datapoints, cannot find your fix for the mentioned typo here

**Review by nayang-cognite — CHANGES_REQUESTED** [review]

**Review by nayang-cognite — APPROVED** [review]

**Pull request merged**